### PR TITLE
[ADD] point_of_sale_enterprise_remove

### PR DESCRIPTION
**Pull request by woodbrettm**
This module removes any settings marked as enterprise from the point_of_sale res_config view.

Cheers,
Brett

@lasley

### CODE REVIEW

**Comment by lasley** [comment]
👍

File: point_of_sale_enterprise_remove/views/res_config_view.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    <record id="view_sale_config_settings_form_pos" model="ir.ui.view">
+        <field name="name">pos.config.settings.form</field>
+        <field name="model">pos.config.settings</field>
+        <field name="inherit_id" ref="point_of_sale.view_sale_config_settings_form_pos"/>
+        <field name="arch" type="xml">
+            <xpath expr="//label[@for='module_pos_loyalty']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//div[field[@name='module_pos_loyalty']]/.." position="attributes">
```
Comment (elicoidal) [inline point_of_sale_enterprise_remove/views/res_config_view.xml:18]: isnot there the restaurant as well?

**Comment by pedrobaeza** [comment]
Remove the SVG file squashing the commit

File: point_of_sale_enterprise_remove/views/res_config_view.xml (hunk 1 above)
Comment (woodbrettm) [inline point_of_sale_enterprise_remove/views/res_config_view.xml:18]: hmmm I don't see an option for that in the res_config view :. Only the loyalty option.

Comment (lasley) [inline point_of_sale_enterprise_remove/views/res_config_view.xml:18]: Looks like Restaurant is Community - https://github.com/odoo/odoo/tree/9.0/addons/pos_restaurant

Comment (elicoidal) [inline point_of_sale_enterprise_remove/views/res_config_view.xml:18]: My bad, I thought pos_restaurant was in enterprise :/
:+1:

Comment (lasley) [inline point_of_sale_enterprise_remove/views/res_config_view.xml:18]: I didn't even know it existed until now, so I'm admittedly glad you made the comment 😄

**Comment by gurneyalex** [comment]
@bmw95 the 9.0 travis build was recently fixed, can you rebase your PR?

**Comment by woodbrettm** [comment]
@pedrobaeza @gurneyalex The 3 PR's have been rebased and are ready to go. The only thing is that in this pr, and crm, the test coverage has been decreased by 0.2% for some reason. The website pr seems good, but a little puzzling why decrease of 0.2% for the other two. Will that be an issue?

**Comment by gurneyalex** [comment]
:+1:

**Comment by woodbrettm** [comment]
As per #499 this pr will be closed.

**Pull request closed without merging**